contexts/SubscriptionContext/src/UseCases/AddSubscription/SubscriptionRequest.php

```diff
@@ -92,6 +94,22 @@ public function setWikilogin( bool $wikilogin ) {
 		$this->wikilogin = $wikilogin;
 	}
 
+	public function getTracking(): string {
+		return $this->tracking;
```

Comment:
I'm not very fond of this name. "What is a tracking" is not even correct English. "tracking" is some kind of purpose/context, with the actual noun missing. `trackingString` seems better, and apparently we're already using it at some places https://github.com/wmde/FundraisingFrontend/search?utf8=%E2%9C%93&q=trackingstring. Might be something better still tho